java-snapshot-testing-core/src/test/java/au/com/origin/snapshots/SnapshotHeaders.java

```diff
@@ -55,5 +55,6 @@ public Snapshot apply(Object object, SnapshotSerializerContext snapshotSerialize
       snapshotSerializerContext.getHeader().put("custom2", "anything2");
       return super.apply(object, snapshotSerializerContext);
     }
-  };
+  }
+  ;
```

Comment:
This is failing format check `./gradlew spotlessApply`

Reply:
`./gradlew spotlessApply` has done this. I was also surprised while commiting.